packages/nextjs/app/builders/0x2d90C8bE0Df1BA58a66282bc4Ed03b330eBD7911/page.tsx

```diff
@@ -0,0 +1,149 @@
+"use client";
+
+import React from "react";
```

Comment:
You don't need to explicitly import React in Next.js anymore, starting from Next.js 12.2 and React 17 and above. With these versions, Next.js automatically imports React when needed, so you can safely omit import React from "react" at the top of your files.